lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -314,7 +320,16 @@ define((require, exports, module) ->
           return
 
         if LatexParsingContext.isType(token, "cite")
-          if @citationsUrl? then @citeCache.getReferences(@citationsUrl, callback)
+          if @citationsUrl? then @citeCache.getReferences(@citationsUrl, (err, cites) =>
```

Comment:
Can you write a comment for these tricks? Also, if we have no `filesUrl` for whatever reason, we won't show any completions at all, right?